packages/grafana-ui/src/utils/standardEditors.tsx

```diff
@@ -191,22 +193,19 @@ export const getStandardFieldConfigs = () => {
     getItemsCount: value => (value ? value.length : 0),
   };
 
-  // const color: FieldConfigPropertyItem<any, string, StringFieldConfigSettings> = {
-  //   id: 'color',
-  //   path: 'color',
-  //   name: 'Color',
-  //   description: 'Customise color',
-  //   editor: standardEditorsRegistry.get('color').editor as any,
-  //   override: standardEditorsRegistry.get('color').editor as any,
-  //   process: identityOverrideProcessor,
-  //   settings: {
-  //     placeholder: '-',
-  //   },
-  //   shouldApply: () => true,
-  //   category: ['Color & thresholds'],
-  // };
-
-  return [unit, min, max, decimals, displayName, noValue, thresholds, mappings, links];
+  const color: FieldConfigPropertyItem<any, FieldColor | undefined, {}> = {
+    id: 'color',
+    path: 'color',
+    name: 'Color',
+    description: 'Define how field/series color is determined',
+    editor: standardEditorsRegistry.get('color').editor as any,
```

Comment:
this should be `colorMode` not color -- I would expect `color` to pick an actually color like in: #24833